doc/fork_support.md

```diff
@@ -0,0 +1,49 @@
+# Background #
+
+In Python multithreading is ineffective at concurrency for CPU bound tasks
+due to the GIL.
```

Comment:
Is it worth stating that it can be effective only in small cases -- when CPU bound tasks explicitly release the GIL?

Reply:
This is true, but it is only possible to release the GIL from an extension module (not pure python).  I've added some details.